lib/spack/spack/installer.py

```diff
@@ -1464,8 +1534,8 @@ def install(self, **kwargs):
                 continue
 
             # Determine state of installation artifacts and adjust accordingly.
-            self._prepare_for_install(task, keep_prefix, keep_stage,
-                                      restage)
+            # Possession of a read lock is required/assumed.
```

Comment:
Given that a read lock was just acquired, IMO this comment is unnecessary.

Reply:
Removing.